script/seed.js

```diff
@@ -2,17 +2,118 @@
 
 const db = require('../server/db')
 const {User} = require('../server/db/models')
+const {Product} = require('../server/db/models')
+const {Category} = require('../server/db/models')
+
+const products = [
+  {
+    name: 'Panda mug',
+    description: 'a mug with cute panda on it',
+    imageUrl: '/images/mug.jpeg',
+    stock: 15,
+    price: 5,
+    },
+  {
+    name: 'Panda at the back',
+    description: 'backpack',
+    imageUrl: '/images/bag.jpg',
+    stock: 10,
+    price: 35,
+  },
+  {
+    name: 'Panda family',
+    description: 'pandy family toys',
+    imageUrl: '/images/toys.jpg',
+    stock: 5,
+    price: 15,
+  },
+  {
+    name: 'Cute Keychain',
+    description: 'keychain',
+    imageUrl: '/images/keychain.jpg',
+    stock: 4,
+    price: 9,
+  },
+  {
+    name: 'Panda on my phone',
+    description: 'phonecase',
+    imageUrl: '/images/phonecase.jpg',
+    stock: 8,
+    price: 16,
+  },
+  {
+    name: 'Panda umbrella',
+    description: 'umbrella',
+    imageUrl: '/images/umbrella.jpg',
+    stock: 1,
+    price: 14,
+  },
+  {
+    name: 'Comfy panda',
+    description: 'pillow',
+    imageUrl: '/images/pillow.jpg',
+    stock: 6,
+    price: 20,
+  }
+]
+
+const categories = [
+    {
+      name: 'home'
+    },
+    {
+      name: 'accesories'
+    },
+    {
+      name: 'kitchen'
+    },
+    {
+      name: 'toys'
+    },
+]
+
+const users = [
+    {
+        isAdmin: true,
+        email: 'cody@email.com',
+        userName: 'codythedog',
+        password: 123,
+        firstName: 'Cody',
+        lastName: 'Dog',
+        billingAddress: '100 Superior st, Chicago, IL'
+    },
+    {
+        email: 'meow@amail.com',
+        userName: 'meowthecat',
+        password: 'abc',
+        firstName: 'Meow',
+        lastName: 'Cat',
+        billingAddress: '200 Super st, Chicago, IL'
+    },
+]
+
+const productCategories = [
+    {productId: 1, categoryId: 3},
+    {productId: 2, categoryId: 2},
+    {productId: 3, categoryId: 4},
+    {productId: 4, categoryId: 2},
+    {productId: 5, categoryId: 2},
+    {productId: 6, categoryId: 1},
+    {productId: 6, categoryId: 2},
+    {productId: 7, categoryId: 1},
+]
+
+
 
 async function seed() {
   await db.sync({force: true})
   console.log('db synced!')
 
-  const users = await Promise.all([
-    User.create({email: 'cody@email.com', password: '123'}),
-    User.create({email: 'murphy@email.com', password: '123'})
-  ])
+  await Product.bulkCreate(products);
+  await Category.bulkCreate(categories);
+  await ProductCategories.bulkCreate(productCategories);
```

Comment:
following our db model naming conventions we're sticking with singular names, so it would be 
```suggestion
  await ProductCategory.bulkCreate(productCategories);
```